### DISCUSSION TITLE
How do I see the output of the `/showIncludes` VC++ compiler flag?

### DISCUSSION THREAD

**Discussion by frank-weindel** [microsoft/vscode-cmake-tools#3095] [Q&A]
I've applied this flag via my CMakeLists.txt, but I don't seem to be able to see the output this flag displays anywhere. Am I missing something?

**Comment by gcampbell-msft** [comment]
@frank-weindel How did you apply the flag via your CMakeLists.txt? 

This discussion may have some context? https://discourse.cmake.org/t/showincludes-option-added-automatically/6706
